### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by vpetrovykh**
I've added more detailed examples for several things in the standard library. In particular I've added more detailed examples of odd behavior of `cal::relative_duration`.

Also, I've added tests for corner cases of `cal::relative_duration` arithmetic.

### CODE REVIEW

**Comment by 1st1** [comment]
@vpetrovykh please fix the ci

File: docs/graphql/limitations.rst [hunk 1]
```diff
@@ -9,8 +9,10 @@ Known limitations
   represented as GraphQL identifiers (e.g. ```N/A``` or ```NOT
   APPLICABLE```) cannot be properly reflected into GraphQL enums.
 
-- EdgeDB :eql:type:`tuples <std::tuple>` are not supported in GraphQL
-  reflection currently.
+- EdgeDB :eql:type:`tuples <std::tuple>` are reflected as either
+  ``List`` or ``Object`` depending on whether they are unnamed or
+  named, respectively. However, currently mutations are not supported
+  for them.
```
Comment (1st1) [inline docs/graphql/limitations.rst:15]:
```suggestion
  named, respectively. However, inserting or updating tuples is
  not yet supported.
```

File: docs/stdlib/datetime.rst [hunk 2]
```diff
@@ -369,47 +370,89 @@ EdgeDB stores and outputs timezone-aware values in UTC.
...
+      ...        <cal::relative_duration>"1 day" +
+      ...        <cal::relative_duration>"1 month";
+      {<cal::local_datetime>'2021-06-01T15:00:00'}
 
     **Gotchas**
 
     Due to the implementation of ``relative_duration`` logic, arithmetic
-    operations may have unintuitive properties. For simplicity, examples below
-    are represented with pseudocode.
+    operations may have unintuitive properties.
```
Comment (1st1) [inline docs/stdlib/datetime.rst:401]:
```suggestion
    operations may behave counterintuitively.
```

?

**Review by 1st1 — APPROVED** [review]

**Review by 1st1 — APPROVED** [review]

**Comment by 1st1** [comment]
@colinhacks Colin can you take a quick look?

**Review by colinhacks — APPROVED** [review]
lgtm!
